build/localnet/Tiltfile

```diff
@@ -199,3 +200,8 @@ k8s_resource(
     port_forwards=["42000:3000"],
     labels=["monitoring"]
 )
+
+test_go('e2e-tests', '../../e2e/tests', '.',
```

Comment:
Some of the other functions in this file accept a `root_dir` and then append the extension. Thoughts on a similar approach?

Reply:
@okdas for a second pair of 👀